modules/ROOT/pages/_partials/data-sync/realtime-intro.adoc

```diff
@@ -0,0 +1,30 @@
+[id="realtime-intro-{context}"]
+= Introduction to real-time updates
+
+After you have developed some queries and mutations, you might want to implement real-time updates.
+These are supported in the GraphQL specification by an operation type called `Subscription`.
+To support scalable, reliable subscriptions in a production environment, Data Sync implements subscriptions using an MQTT PubSub subscription mechanism, however you might want to use the Apollo PubSub module to develop proof-of-concept applications.  
+
+When coding for real-time updates, you use the following modules:
+
+* @aerogear/voyager-server - supports clients that use voyager-client to enable GraphQL queries and mutations
+* @aerogear/voyager-subscriptions - supports clients that use voyager-client to enable GraphQL subscriptions
+* @aerogear/graph-mqtt-subscriptions - supports GraphQL resolvers connections to a MQTT broker
+
+GraphQL Subscriptions utilise websockets to enable clients to subscribe to published changes.
```

Comment:
I might suggest something like `GraphQL Subscriptions enable clients to subscribe to events <generated within|that happen on> the server over a websocket connection`

Reply:
updated